logstash-core/src/main/java/org/logstash/common/io/PageIO.java

```diff
@@ -10,10 +10,20 @@
 public interface PageIO extends Closeable {
 
     // the concrete class should be constructed with the pageNum, capacity and dirPath attributes
+    // and open/recover/create must first be called to setup with physical data file
+    //
+    // TODO: we should probably refactor this with a factory to force the creation of a fully
```

Comment:
oh, ok. I just didn't see this TODO copy/pasted from anywhere, so I assumed this was a related note